packages/system/src/components/homepage/contributor-banner.tsx

```diff
@@ -33,8 +33,16 @@ export function ContributorBanner({
 					Create a PR if you see mistakes, room for improvement, or new
 					opportunities to help the dev team.
 				</p>
+				<Button
+				color="primary"
+				size="large"
+				as="a"
+				href="https://github.com/thisdot/framework.dev"
+			>
+				Contribute
+			</Button>
```

Comment:
Also, the button design does not match the design [laid out here](https://github.com/thisdot/framework.dev/issues/481#issuecomment-1335922647)

The button width needs to be wider and there should be some margin top so there is space between the button and the paragraph.